src/foam/nanos/logger/DAOLogger.js

```diff
@@ -76,7 +76,10 @@ if ( initializing.get() ) {
   System.out.println("DAOLogger initializing");
   return;
 }
-LogMessage lm = new LogMessage.Builder(getX()).setSeverity(severity).setMessage(message).build();
+LogMessage lm = new LogMessage.Builder(getX())
+                    .setThreadName(String.valueOf(Thread.currentThread().getId()))
```

Comment:
Is this necessary? Shouldn't the factory do this for you?

Reply:
yes. updated.